spinalcordtoolbox/scripts/sct_compute_compression.py

```diff
@@ -50,16 +50,21 @@ def get_parser():
                     'from healthy control subjects. This database uses the PAM50 template as an anatomical reference '
                     'system.\n'
                     '\n'
-                    'Reference:\n'
+                    'References:\n'
+                    'Sandrine Bédard, Jan Valošek, Maryam Seif, Armin Curt, Simon Schading, Nikolai Pfender, Patrick '
+                    'Freund, Markus Hupp, Julien Cohen-Adad. Normalizing Spinal Cord Compression Morphometric '
+                    'Measures: Application in Degenerative Cervical Myelopathy. medRxiv 2024.03.13.24304177\n'
+                    'doi.org/10.1101/2024.03.13.24304177\n'
+                    '\n'
                     'Miyanji F, Furlan JC, Aarabi B, Arnold PM, Fehlings MG. Acute cervical traumatic '
                     'spinal cord injury: MR imaging findings correlated with neurologic outcome--prospective study '
                     'with 100 consecutive patients. Radiology 2007;243(3):820-827.\n'
                     'doi.org/10.1148/radiol.2433060583\n'
                     '\n'
                     'Reference for the "-normalize-hc" flag:\n'
                     'Valošek J, Bédard S, Keřkovský M, Rohan T, Cohen-Adad J. A database of the healthy human spinal '
-                    'cord morphometry in the PAM50 template space. NeuroLibre Reproducible Preprints 2023; 17.\n'
-                    'doi.org/10.55458/neurolibre.00017'
+                    'cord morphometry in the PAM50 template space. Imaging Neuroscience 2024; 2 1–15.\n'
+                    'doi.org/10.1162/imag_a_00075'
```

Comment:
Some small tweaks:

- Add bullet points to improve indentation auto-formatting:
- Add "https://" to the links so that they'll show up as links in the terminal (and so that they will be checked by the broken link checker)

```python
                    'References:\n'
                    '  - Sandrine Bédard, Jan Valošek, Maryam Seif, Armin Curt, Simon Schading, Nikolai Pfender, '
                    'Patrick Freund, Markus Hupp, Julien Cohen-Adad. Normalizing Spinal Cord Compression Morphometric '
                    'Measures: Application in Degenerative Cervical Myelopathy. medRxiv 2024.03.13.24304177\n'
                    '    https://doi.org/10.1101/2024.03.13.24304177\n'
                    '  - Miyanji F, Furlan JC, Aarabi B, Arnold PM, Fehlings MG. Acute cervical traumatic '
                    'spinal cord injury: MR imaging findings correlated with neurologic outcome--prospective study '
                    'with 100 consecutive patients. Radiology 2007;243(3):820-827.\n'
                    '    https://doi.org/10.1148/radiol.2433060583\n'
                    '  - "-normalize-hc" flag:\n'
                    '    Valošek J, Bédard S, Keřkovský M, Rohan T, Cohen-Adad J. A database of the '
                    'healthy human spinal cord morphometry in the PAM50 template space. Imaging Neuroscience 2024; '
                    '2 1–15.\n'
                    '    https://doi.org/10.1162/imag_a_00075'
```

Result: 

![image](https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/16181459/89a69bc9-adbe-41ea-a9c8-9f8a2177715a)

Reply:
Good idea! Agree!